### PR TITLE
OpenEdge plugin: upgrade to 2.24.2

### PR DESCRIPTION
**Pull request by gquerret**
Version 2.24.2 has been released two weeks ago, could you please update the marketplace ?
SonarCloud project: https://sonarcloud.io/summary/new_code?id=eu.rssw%3Asonar-openedge

Thank you !

### CODE REVIEW

**Comment by gquerret** [comment]
Does the build failure have anything to do with my change ?

File: openedge.properties [hunk 1]
```diff
@@ -18,3 +18,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.19.2.date=2023-01-10
```
Comment (ganncamp) [inline openedge.properties:18]: This version can't be left compatible with `LATEST`; only one version can be marked compatible with `LATEST` at a time. Set the end of this version's range to `10.3` (or `10.3.*`)

File: openedge.properties [hunk 2]
```diff
@@ -18,3 +18,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
...
 2.19.2.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.19.1
 2.19.2.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.19.1/sonar-openedge-plugin-2.19.2.jar
+
+2.24.2.description=Version 2.24.2
+2.24.2.sqVersions=[8.9,LATEST]
```
Comment (ganncamp) [inline openedge.properties:23]: This range needs to start with the current LTS: `9.9`

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: openedge.properties (hunk 1 above)
Comment (gquerret) [reply openedge.properties:18]: Changed to 9.9.* (this version was not tested with 10.x)

File: openedge.properties (hunk 2 above)
Comment (gquerret) [reply openedge.properties]: Changed

**Review by ganncamp — APPROVED** [review]

**Comment by ganncamp** [comment]
FYI, I didn't look at it, but the build failure was likely about configuration on our side. We recently moved where the production job for this runs, and likely didn't also update the test build.

**Comment by antoine-vigneau-sonarsource** [comment]
Hello @gquerret,

It appears that your [Version 2.24](https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.24.1) got re-released and then _sonar-openedge-plugin-2.24.2.jar_ is no longer available in favor of _sonar-openedge-plugin-2.24.3.jar_

Consequently, our job fails because it can't find _sonar-openedge-plugin-2.24.2.jar_ 🚨

I'm sure you'd have open a PR to propagate this change in our repo here. I did it myself to act fast on the build failure (#497 ).
But generally speaking, wouldn't it make sense to create a new release rather than modifying an existing one? That will avoid any automation failure.

cc @ganncamp

**Comment by gquerret** [comment]
@antoine-vigneau-sonarsource Hotfix 2.24.3 of this plugin was released in a hurry, and I didn't realize that modifying the release instead of creating a new one would break the update center. I'll open a new PR with the latest version.

**Pull request merged**